services/user.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.models import User
+
+def create_user(username, password, email=None, first_name=None, last_name=None):
+    user = User.objects.create_user(username, email, password)
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
+
+def get_user(user_id):
+    return User.objects.get(id=user_id)
```

Comment:
The `get_user` function does not handle the case where a user with the specified ID does not exist. Consider using `get_object_or_404` or handling the `User.DoesNotExist` exception to provide a more user-friendly error handling.